packs/classfeatures/divine-font.json

```diff
@@ -24,7 +24,27 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "img": "icons/magic/life/cross-worn-green.webp",
+                        "label": "PF2E.SpecificRule.Cleric.DivineFont.Healing",
+                        "value": "heal"
+                    },
+                    {
+                        "img": "systems/pf2e/icons/spells/harm.webp",
+                        "label": "PF2E.SpecificRule.Cleric.DivineFont.Harmful",
+                        "value": "harm"
+                    }
+                ],
+                "flag": "divineFont",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Cleric.DivineFont.Prompt",
+                "rollOption": "self:divine-font"
```

Comment:
Is there a reason to use a self domain here?

Reply:
No specific reason.